roscpp_azure_iothub/README.md

```diff
@@ -68,18 +68,25 @@ catkin_make install
 source /install/setup.bash
 ```
 
-# Deployment (IoT Hub for telemetry reporting)
-Device twins are JSON documents that store device state information including metadata, configurations, and conditions. Azure IoT Hub maintains a device twin for each device that you connect to IoT Hub. And we are using `desired` properties as a channel to ask our ROS node what ROS topics to report.
+# Deployment (IoT Hub for telemetry reporting or reported properties channel)
+Device twins are JSON documents that store device state information including metadata, configurations, and conditions. Azure IoT Hub maintains a device twin for each device that you connect to IoT Hub. And we are using `desired` properties as a channel to ask our ROS node what ROS topics to report. The messages recieved from these ROS topics can either be reported through the telemetry channel or through the `reported` property channel. 
```

Comment:
Why would you want to use reported verses telemetry?
Should these actually be a property on the twin instead of a message relay?

Reply:
The telemetry channel is designed for high bandwidth data and can sometimes be overwhelming because the history of the messages sent over the channel are stored. On the other hand, reported properties is good for low bandwidth data and advertises the latest message (good if you don't care about past messages) and you always know that you have the latest data on hand. From the app side, using reported properties is easier because the telemetry channel would require more integration just to get the latest message. Should I add these differences in the readme?

Reply:
Cool. Please add this to the readme.